README.md

```diff
@@ -1,16 +1,73 @@
-# Title
+# JavaScript Class Extension Lab: `super`
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+1. Practice building additional functionality into subclasses
 
 ## Introduction
 
-## SWBAT 1
+In this lab we're going to create a geometry application that allows us to
+calculate different properties of multiple shape. We'll be using JavaScript's
+`class` keyword to build objects that inherit from its base object.
 
-## SWBAT 2
+## Practice Building Additional Functionality Into Subclasses
 
-## Conclusion
+1. Define a `Quadrilateral` class that accepts *four* integer parameters.
+* Define a `saturation` property set as `255` that will be inherited by subclasses.
+* Define a `perimeter()` method that calculates perimeter.
+* Define a conditional `color()` method that checks for red, green, blue, yellow, cyan, magenta, and black. It returns a `String` value of the color name (eg: "red").
+
+2. Define a `Square` subclass.
+* Define a static `name` property of `square`.
+* Square can calculate perimeter
+* Define a setter method that takes `saturation` and sets RGB to 255,0,0 (red)
+* Define a method called `sayColor()` on `Square` that returns "The square is red" that uses the method from `color()` and the `name` property.
+* Define a method called `description()` on `Square` that returns: "A parallelogram in which all the sides are equal and each angle measures 90° is called a square."
+
```

Comment:
I think these are awesome examples. I think though I will wait until the solution and tests are up to check through these